app/src/main/java/de/tum/in/tumcampusapp/api/app/TUMCabeAPIService.java

```diff
@@ -209,6 +211,10 @@
 
     // TICKET SALE
 
+    // openingHours\
+    @GET("Api/openingtimes")
+    Observable<OpeningHours.Result> displayCategory(@Query("category") String category);
```

Comment:
There is not `category` filter on this endpoint. Please just download the whole file once and do any filtering on the clientside. Cache should be efficient enough, to not make this a problem.